mlton/ssa/ssa-tree.sig

```diff
@@ -121,6 +121,7 @@ signature SSA_TREE =
              | Tuple of Var.t vector
              | Var of Var.t
 
+	    val size: t -> int
```

Comment:
Values in signature should be in alphabetical order; should go after `replaceVar`.